### ISSUE TITLE
About training steps

### ISSUE THREAD

**Issue by 123dddwn** [zty557/TADSR#6]
Thanks for your excellent work. But I would like to know why the results are not good when using the parameter settings described in your paper. Can images really be restored well with only 2,000 iterations?

**Comment by zty557** [comment]
Thank you for your question. We did train for 2k epochs and successfully reproduced the results reported in the paper. We have also provided the corresponding checkpoint at [Hugging Face - TADSR Checkpoint](https://huggingface.co/zty557/TADSR)
. If you are unable to achieve similar performance, the issue may be related to the dataset or training hardware/setup. You are welcome to provide more detailed information so that we can further discuss and troubleshoot the problem together.

**Comment by nitec427** [comment]
Hi @zty557, and thanks for the great work! I was wondering whether “2K epochs” in your setup means 2,000 full training epochs or 2,000 gradient update iterations. I’d like to try reproducing your setup, so I’m also trying to estimate the total training time.
